Source/Csla/Server/ActiveAuthorizer.cs

```diff
@@ -0,0 +1,75 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ActiveAuthorizer.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Implementation of the authorizer that</summary>
+//-----------------------------------------------------------------------
+using Csla.Security;
+
+namespace Csla.Server
+{
+  /// <summary>
+  /// Implementation of the authorizer that
+  /// checks per-type authorization rules
+  /// for each request.
+  /// </summary>
+  public class ActiveAuthorizer : IAuthorizeDataPortal
+  {
+    /// <summary>
+    /// Creates an instance of the type.
+    /// </summary>
+    /// <param name="applicationContext"></param>
+    public ActiveAuthorizer(ApplicationContext applicationContext)
+    {
+      ApplicationContext = applicationContext;
+    }
+
+    private ApplicationContext ApplicationContext { get; set; }
+
+    /// <summary>
+    /// Checks authorization rules for the request.
+    /// </summary>
+    /// <param name="clientRequest">
+    /// Client request information.
+    /// </param>
+    public void Authorize(AuthorizeRequest clientRequest)
```

Comment:
Should this only be applied when the location and logical location values are Server, so that it doesn't run when the data access operation is run locally, such as with [RunLocal]?